docs/content/reference/api/github.com/solo-io/gloo/projects/gloo/api/v1/enterprise/options/extauth/v1/extauth-internal.proto.sk.md

```diff
@@ -1,14 +1,14 @@
 
 ---
-title: "extauth-internal.proto"
+title: "ExtauthInternal"
```

Comment:
We're good with dashes being treated the same as underscores in titles, right?
(Long term we might want to omit this file entirely.)

Reply:
I think this is ok. This is purely a visual effect (ie the title) and I think (given Nadine's approval) this is ok

Reply:
If @Nadine2016 is cool with this, so am I!

Reply:
This is fine from my end. Thank you for checking in. @arianaw66 @sam-heilbron